README.md

```diff
@@ -121,37 +121,48 @@ clients for an example.
+If the app is paired via the Pair App dialog, the QR code that will be scanned contains a short-lived URL, e.g. `https://radar-base-url.org/api/meta-token/bMUkowOmTOci`
+
+Your app should access the URL, where it will receive an OAuth2 
+refresh token as well as the platform's base URL and a URL to the privacy policy. No authorization 
+is required to access this URL. **Important:** For security reasons, the information at this URL can
+only be accessed once. Once it has been accessed it can not be retrieved again. 
+
+The app can use that refresh token to get new access and refresh tokens by doing the following HTTP 
+request to the base URL, using HTTP basic authentication with your OAuth client ID as username, and 
+an empty password.
 ```
-POST MyId:MySecret /oauth/token
+POST /oauth/token
 Content-Type: application/x-www-form-urlencoded
 
-grant_type=refresh_token&refresh_token=...
+grant_type=refresh_token&refresh_token=<refresh_token>
 ```
-This will respond with the access token and refresh token:
+This will respond with at least the access token and refresh token:
 ```json
 {
    "access_token": "...",
    "refresh_token": "...",
+   "expires_in": 14400
 }
 ```
-For the next request, you need to use the `refresh_token` that was returned rather than the one in the QR code, since refresh tokens are valid only once.
+Both tokens are valid for a limited time only. When the access token runs out, you will need to 
+perform another request like the one above, but you need to use the new `refresh_token`, since 
+refresh tokens are valid only once.
 
 The code grant flow for OAuth2 clients can be the following:
 1. Ask user confirmation for your app:
      ```
-     GET /oauth/confirm_access?client_id=MyId&client_secret=MySecret&grant_type=code&redirect_uri=https://my.example.com/oauth_redirect
+     GET /oauth/confirm_access?client_id=MyId&grant_type=code&redirect_uri=https://my.example.com/oauth_redirect
      ```
-     This needs to be done from a interactive web view, either a browser or a web window. If the user approves, this will redirect to `https://my.example.com/oauth_redirect?code=abcdef`. In Android, with [https://appauth.io](AppAuth library), the URL could be `com.example.my://oauth_redirect` for the `com.example.my` app.
-2. Request a token for you app:
+     where you replace `MyId` with your OAuth client id. This needs to be done from a interactive 
+     web view, either a browser or a web window. If the user approves, this will redirect to 
+     `https://my.example.com/oauth_redirect?code=abcdef`. In Android, with [https://appauth.io]
+     (AppAuth library), the URL could be `com.example.my://oauth_redirect` for the `com.example.my`
+      app.
+2. Request a token for your app by doing a POST, again with HTTP basic authentication with as 
+username your OAuth client id, and leaving the password empty:
     ```
-    POST MyId:MySecret /oauth/token
+    POST /oauth/token
```

Comment:
Doesn't this need to include the client ID as a username, e.g. `MyId@/oauth/token`?